services/cd-service/Dockerfile

```diff
@@ -1,7 +1,30 @@
+ARG PARENT_CONTAINER
+FROM ${PARENT_CONTAINER} as builder
```

Comment:
In `docker-compose.yaml` the `PARENT_CONTAINER` arg is set to `latest` which is an incorrect name for the base image.